core/interfaces.go

```diff
@@ -11,9 +11,9 @@ import (
 	"net/http"
 	"time"
 
-	jose "github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
```

Comment:
I did.  I agree that it can sometimes be confusing, but the fact is that the mnemonic here is redundant information.  ISTM that we should set a rule here.  How about we set an explicit name when (1) it's needed for conflict resolution (as with `blog`), or (2) the package name deviates from the "last path segment" convention (as with `jose`).  Either way, I would propose that we do the same thing for a given package regardless of where it is imported.